api-reference/auth/desktop-sdks/web.mdx

```diff
@@ -28,20 +32,20 @@ yarn add @particle-network/auth-core-modal
 
 ## Configuration
 
-Before interacting with Particle, you'll need to import and configure your master configuration object; this'll be used to not only customize the embedded wallet modal which is optionally shown after logging in, but also will allow you to define required keys and authenticate your project.
-
-Although before we import and configure this object, you'll need to retrieve the aforementioned keys.
+The main configuration is set up by wrapping the application with a master configuration object. This object is essential for authenticating your project and customizing the embedded wallet modal, which can be optionally displayed after login.
```

Comment:
Configuration is primarily handled by wrapping an application with Particle Auth's master configuration component; this is essential for authenticating your project and [customizing the optional embedded wallet modal](/guides/configuration/appearance/wallet).